src/Microsoft.DotNet.Tools.Builder/README.md

```diff
@@ -0,0 +1,32 @@
+dotnet-build
+===========
+
+**NAME** 
+dotnet-build -- Orchestrates the compilation of a project and all its dependencies.
+
+**SYNOPSIS**
+dotnet build[options]
+
+**DESCRIPTION**
+
+The build verb orchestrates the compilation of a project: it gathers the dependencies of a project and decides which to compile. 
+
+Users should invoke the Build verb when they want the entire dependency graph compiled, and Compile when they want only a specific project compiled.
```

Comment:
Will incremental support be available when "dotnet compile" is used to compile a single project?